### DISCUSSION TITLE
"Dead Man's switch" functionality?

### DISCUSSION THREAD

**Discussion by mynamewastaken** [keepassxreboot/keepassxc#6166] [Ideas]
Something similar to https://support.dashlane.com/hc/en-us/articles/360008918919-Introduction-to-the-Emergency-feature

**Comment by phoerious** [comment]
Since KeePassXC is entirely offline and basically just an encrypted file on your hard drive, I'm not quite sure how something like this should work. The way you would do it is to give them the password in advance.

**Reply by mynamewastaken** [reply]
I suspected this but wanted to bring it up in case there were any plans for it or prior issues/discussion (I couldn't find anything relevant).

Thanks for the quick reply!

**Comment by nagromc** [comment]
You can split your master password into chunks with the [Shamir's Secret Sharing](https://en.wikipedia.org/wiki/Shamir%27s_Secret_Sharing), and share them to your family members and friends.

The goal is that your can split your master password into x chunks, but you can decide that only y of them will be needed to recover it, where y ⩽ x.

There are several implementations:
- Debian: https://packages.debian.org/buster/ssss
- Online version: https://simon-frey.com/s4/
- And probably others…

**Reply by phoerious** [reply]
No, you can't. https://en.bitcoin.it/wiki/Shamir_Secret_Snakeoil
